### ISSUE TITLE
`endpoints.get_current_user()` does not support 3rd-party auth

### ISSUE THREAD

**Issue by lookuptable** [cloudendpoints/endpoints-python#53]
Currently the `get_current_user()` method returns a user object only if the user is created from a Google ID token or an opaque OAuth token. We should fix it so that it works for users from a 3rd-party auth provide.

**Comment by presveva** [comment]
seems related to #32

**Comment by Al77056** [comment]
This is fixed now with version 2.0.5

**Comment by presveva** [comment]
@Al77056 thank you.

**Comment by presveva** [comment]
@Al77056 No, with the 2.0.5 `endpoints.get_current_user()` returns the google user, the same obj of `google.appengine.api.users.get_current_user()`, which have nothing to do with the 3rd-party user obj (firebase in my case).

**Comment by tangiel** [comment]
@presveva it's not meant to return a third party SDK object, rather user info decoded from the JWT.
